### ISSUE TITLE
Tooltip responsive positioning

### ISSUE THREAD

**Issue by jpante** [Workday/canvas-kit#1100]
## 🐛 Bug Report

The tooltips do not position/wrap according to the space on the screen

## To Reproduce

1. Change the modules/tooltip/react/stories/CustomContent tooltip:
2. center the tooltip
3. placement to be 'left'
4. Open storybook using developer environment
5. Shrink the browser down so there is minimum available space between the button and the edge of the browser. The text content of the tooltip should span longer than the width of available space.
6. Hover on the button and notice the tooltip does not have enough space on the left or right side to properly display and is cut off.

## Expected Behavior

The tooltips should position according to the space on the screen

## Actual Results

The tooltips does not position according to the space on the screen.
On IE11, the long tooltip does not  fit within the viewport

### Error Output

![image](https://user-images.githubusercontent.com/1861673/121414634-b3a27300-c91b-11eb-8b0d-62c872a21331.png)

**Comment by jaclynjessup** [comment]
Cannot repro issue today, we also no longer support IE11. Closing as we will not fix.

**Comment by NicholasBoll** [comment]
This issue is not fixed and is a problem on all browsers

**Comment by thunguyen19** [comment]
Is this still a valid bug? I cannot reproduce on Chrome.

**Comment by jaclynjessup** [comment]
@thunguyen19 I will sync with you directly about this issue.
